### PR TITLE
Fixed : great people could not build improvement.

### PR DESCRIPTION
**Pull request by ninjatao**

### CODE REVIEW

File: core/src/com/unciv/ui/pickerscreens/ImprovementPickerScreen.kt [hunk 1]
```diff
@@ -48,6 +48,7 @@ class ImprovementPickerScreen(val tileInfo: TileInfo, val onAccept: ()->Unit) :
         regularImprovements.defaults().pad(5f)
 
         for (improvement in tileInfo.tileMap.gameInfo.ruleSet.tileImprovements.values) {
+            if (improvement.hasUnique("Great Improvement")) continue
```
Comment (yairm210) [inline core/src/com/unciv/ui/pickerscreens/ImprovementPickerScreen.kt:51]: It's the 'unbuildable' that should restrict it from being built, not the 'Great Improvement'

Comment (yairm210) [reply core/src/com/unciv/ui/pickerscreens/ImprovementPickerScreen.kt:51]: Amazing to see another fix from you :D

Comment (ninjatao) [reply core/src/com/unciv/ui/pickerscreens/ImprovementPickerScreen.kt:51]: Done. I am currently a bit too busy to do big features, so some bug fixes from time to time is all I can do. ^_^

**Comment by yairm210** [comment]
We should probably check the worker automation - it might take the list of possible improvements without paying attention to the "Unbuildable"

**Comment by ninjatao** [comment]
> We should probably check the worker automation - it might take the list of possible improvements without paying attention to the "Unbuildable"

Automate worker actions are fine. It chooses improvement based on terrain/resource status.

**Pull request merged**